### PR TITLE
add deprecation notice to README

### PR DESCRIPTION
**Pull request by mvdan**

### CODE REVIEW

**Review by myitcv — APPROVED** [review]

**Review by cueckoo — APPROVED** [review]

**Review by porcuepine — APPROVED** [review]

**Comment by mvdan** [comment]
Needed an approval from porcuepine since we used CODEOWNERS to block direct PR merges in favor of GerritHub. OK now that we're about to archive the repo.

**Pull request merged**
